### DISCUSSION TITLE
سرویس جدید فیلتر شکن گوگل

### DISCUSSION THREAD

**Discussion by AnonymousMaximus2** [WeAreMahsaAmini/Core#402] [Ideas]
گوگل ابزار جدید خود برای دور زدن فیلترینگ در ایران را فعال کرد

شرکت گوگل ابزار جدید خود برای دور زدن فیلترینگ در ایران را فعال کرد.

با ورود به صفحه اصلی گوگل به زبان فارسی، متنی درباره نحوه استفاده از ابزار Outline با هدف ایجاد «وی‌پی‌ان» امن برای کاربران ظاهر می‌شود.

شرکت گوگل قبلا نیز اعلام کرده بود که در پی صدور مجوزهای جدید خزانه‌داری آمریکا، ابزارهای خود برای دور زدن فیلترینگ در ایران را ارائه خواهد داد.

متن کامل خبر را اینجا بخوانید:

https://www.radiofarda.com/a/32042577.html

سایت پروژه گوگل را اینجا ببینید:

https://getoutline.org/fa

پروژه گیتهاب گوگل را اینجا دنبال کنید:

https://github.com/Jigsaw-Code/?q=outline

**Comment by AnonymousMaximus2** [comment]
اگر وب سایت مهسا امینی یا اپلیکیشن سایت با این تکنولوژی گوگل ادغام شود، میتواند برای ایرانیان داخل بسیار مفید باشد.


همچنین میتواند یک منبع درآمدی برای افرادی باشد که این سرویس را نصب و مدیریت میکنند و به سایر افراد داخل ایران میفروشند تا هزینه های تبلیغات و آموزش و اطلاع رسانی خودشان را تامین کنند و بتوانند با قدرت بیشتری فعالیت کنند.

**Comment by sorsaffari** [comment]
کسی از ایران بازخوردی در مورد استفاده از این vpn شنیده؟

**Reply by amirhemmateenejad** [reply]
من نصب کردم روی سرور 
روی اینترنت همراه اول برای من کار کرد ولی روی ایرانسل و اینترنت point to point جواب نداد

**Reply by shabane** [reply]
دلیلش اینه که outline از دو تا پورت خاص استفاده میکنه، باید این دو رو تغییر بدی به مثلا 443 یا 80
که اینقدر مشخص نباشه،

البته یه مسئله دیگه هم هست، اگه اینترنت ایران اکسس بشه، کلا نمیتونی ازش استفاده کنی، یه سرور توی ایران بگیر و پورت 443 اش رو وصل کن به پورت outline توی سرور خارجی

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر

**Reply by AnonymousMaximus2** [reply]
تکبیییر :) 


اون مهدی 313 بزنه توی سرت که توی 1400 سال از زمان اسلام تا الان تعداد 313 نفر ادم پیدا نکردی تا امام زمانت ظهور کنه

**Comment by theStratus** [comment]
اموزش نصب تر تمیزی ازش هست؟
ایا تنظیمات خاصی میخواد که بلند مدت تر کار کنه؟ مثل تغییر پورت

**Reply by shabane** [reply]
بله هست، یه راه اینه که یه سرور توی ایران بگیری و پورت مثلا 80 یا 443 رو به پورت outline روی سرور خارجی تونل کنی،

**Reply by theStratus** [reply]
ممنونم عزیز 
آیا راهنمای تر تمیزی برای نصبش داری؟ اسکریپت های ساده نصب منظورمه. چیزی که بشه پخش کرد و ملت با اطلاعات فنی کم هم بتونن نصب کنن.

**Reply by shabane** [reply]
ببخشید الان جواب می‌دم، پیام رو فراموش کرده بودم، براز درست کردن تونل از ایران اسکریپت ساده براش نیست که انجام بده، اما خیلی ریپو ها هست که اینکار رو انجام می‌ده
